### PR TITLE
[GDPR] More profile data fields

### PR DESCRIPTION
**Pull request by alrocar**
https://github.com/CartoDB/cartodb-central/issues/2184

### CODE REVIEW

File: app/models/carto/user.rb [hunk 1]
```diff
@@ -44,7 +44,8 @@ class Carto::User < ActiveRecord::Base
                    "users.viewer, users.quota_in_bytes, users.database_host, users.crypted_password, " \
                    "users.builder_enabled, users.private_tables_enabled, users.private_maps_enabled, " \
                    "users.org_admin, users.last_name, users.google_maps_private_key, users.website, " \
-                   "users.description, users.available_for_hire, users.frontend_version, users.asset_host".freeze
+                   "users.description, users.available_for_hire, users.frontend_version, " \
+                   "users.asset_host, users.company, users.phone, users.job_role, users.industry".freeze
```
Comment (javitonino) [inline app/models/carto/user.rb]: This probably doesn't need to be added here. This is the list of fields that are needed by the visualization presenter to show the data it needs about the user (to try to make the queries smaller).

Remove it if you can, but don't hesitate to add it back if test break.

File: app/services/carto/user_metadata_export_service.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ module UserMetadataExportServiceConfiguration
       :salesforce_datasource_enabled, :builder_enabled, :geocoder_provider, :isolines_provider, :routing_provider,
       :github_user_id, :engine_enabled, :mapzen_routing_quota, :mapzen_routing_block_price, :soft_mapzen_routing_limit,
       :no_map_logo, :org_admin, :last_name, :user_render_timeout, :database_render_timeout, :frontend_version,
-      :asset_host, :state
```
Comment (javitonino) [inline app/services/carto/user_metadata_export_service.rb]: Please bump the version of the user metadata export service and add a test to check that the new version imports the data generated by the old one (sets the fields to nil without crashing, basically).

**Comment by javitonino** [comment]
Working as expected. Will deploy tomorrow. Can you please split migrations into their own PR?
